spec/europeana/api/record/hierarchy_spec.rb

```diff
@@ -0,0 +1,29 @@
+RSpec.describe Europeana::API::Record::Hierarchy do
+  let(:record) { instance_double("Europeana::API::Record", id: record_id) }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.